app/presenters/instance_presenter.rb

```diff
@@ -46,6 +46,10 @@ def languages
     [I18n.default_locale]
   end
 
+  def supported_languages
+    LanguagesHelper::SUPPORTED_LOCALES
```

Comment:
What's the purpose in including both the international and the native name along with every language code?

Shouldn't this just be `LanguagesHelper::SUPPORTED_LOCALES.keys`?